src/Breadcrumbs/__tests__/Breadcrumbs.test.tsx

```diff
@@ -1,10 +1,8 @@
 import React from 'react'
-import {Breadcrumbs, Breadcrumb} from '..'
-import {render, behavesAsComponent, checkExports} from '../utils/testing'
+import {default as Breadcrumbs, Breadcrumb} from '..'
```

Comment:
This is not related this PR but we seem to use mix of default and named exports - I am not sure if there is any pattern or we do it randomly?